components/climate/ir_climate.rst

```diff
@@ -154,11 +139,63 @@ Configuration variables:
         header_high: 3265us # AC Units from LG in Brazil, for example use these timings
         header_low: 9856us
 
+
+.. _toshiba:
+
+``toshiba`` Climate
+---------------------
```

Comment:
Too many dashes?

Reply:
Too many dashes. I thought the linter catches these...